### DISCUSSION TITLE
Provide info on how to donate to support the project

### DISCUSSION THREAD

**Discussion by apravato** [KilledByAPixel/LittleJS#129] [Ideas]
Hi! I would like to ask the company I work for to start making a one-off donation to sponsor a bit the development of LittleJs and say thank you, but I have not found info about where to send the payment and potential tax-deductibility. It would be comfortable to have these info easily reachable, to reduce friction in contributing economically. Thanks!

**Comment by KilledByAPixel** [comment]
Thank you so much, I am looking into setting up something like a Patreon for LittleJS and all the other stuff that I do. This will be a goal for next year.

Another idea is that my solo company Frank Force Games can help with all kinds of things from advanced consolations and feedback sessions for any type of software to web development and specially made interactive advertisements. If it sounds like there something I could help out with shoot me an email at frankforce@gmail.com and we'll set up a meeting!

- Frank
